INSTALL.md

```diff
@@ -31,6 +31,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_BPF_EVENTS=y
 ```
 
+For Linux kernel version 4.7 or newer, the flag `CONFIG_HAVE_BPF_JIT` has been replaced by `CONFIG_HAVE_EBPF_JIT` (for x86, arm64 and s390) or by `CONFIG_HAVE_CBPF_JIT` (for arm, mips, powerpc and sparc.)
```

Comment:
Anyway, if you think user don't need to consider about status of the `CONFIG_HAVE_` flags, that's fine. Please just close the pull request. 
And I do hope you could delete the `CONFIG_HAVE_BPF_JIT` in existing document due to the same reason. 
Thanks!